packages/forklift-console-plugin/src/modules/Plans/views/list/components/PlanStatusVmCount.tsx

```diff
@@ -32,11 +32,17 @@ export const PlanStatusVmCount: React.FC<PlanStatusVmCountProps> = ({
   }, [status]);
 
   return (
-    <Flex alignItems={{ default: 'alignItemsCenter' }} spaceItems={{ default: 'spaceItemsSm' }}>
+    <Flex
+      alignItems={{ default: 'alignItemsCenter' }}
+      spaceItems={{ default: 'spaceItemsSm' }}
+      data-testid={`plan-status-vm-count-${status}`}
+    >
       <Icon status={status}>{statusIcon}</Icon>
 
       <FlexItem>
-        <Link to={linkPath}>{t('{{total}} VM', { count, total: count })}</Link>
+        <Link to={linkPath}>
+          {count > 99 ? t('99+ VMs') : t('{{total}} VM', { count, total: count })}
```

Comment:
u need count sending to t?

Reply:
Same as https://github.com/kubev2v/forklift-console-plugin/pull/1442#discussion_r1932279697